### DISCUSSION TITLE
Signing the Scintilla.Net assemblies

### DISCUSSION THREAD

**Discussion by HaraldurSam** [desjarlais/Scintilla.NET#84] [Ideas]
Is there any plan on providing signed version of the Scintiall.Net nuget package?

Regards,
Haraldur

**Comment by VPKSoft** [comment]
Not by me as I'm looking for new maintainer for the project. There is an issue open, see #77, but as mentioned in the comment the signed package would just increase complexity of the deployment process, etc. I see no reason to do so.

**Comment by desjarlais** [comment]
I'm going to close this discussion.  I'll keep up the conversation in issue #77 but I think I understand that use case, so this type of discussion probably is no longer needed and we can talk about it in the issue.
